src/main/drivers/accgyro/accgyro_spi_icm20689.c

```diff
@@ -55,7 +55,7 @@ uint8_t icm20689SpiDetect(const busDevice_t *bus)
 {
     icm20689SpiInit(bus);
 
-    spiSetDivisor(bus->busdev_u.spi.instance, SPI_CLOCK_INITIALIZATON); //low speed
+    spiSetDivisor(bus->busdev_u.spi.instance, SPI_CLOCK_INITIALIZATION); //low speed
```

Comment:
lol

Reply:
It took me full 5 minutes or so to find out why it keep throwing errors at me! 🤣